### ISSUE TITLE
Updated logo

### ISSUE THREAD

**Issue by twentysixmoons** [open-austin/open-data-progress-report#22]
We need to update the logo on the progress report to the latest Open Austin logo

**Comment by mateoclarke** [comment]
Good call, I'll commit to working on this along with the website this Civic Hack Saturday.
